src/pages/en/guides/migrate-to-astro.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate an existing site to Astro
+description: Some tips and tricks for converting your site to Astro.
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
+i18nReady: true
+---
+
+SARAH ROUGH NOTES
+
+**Ready to convert your site to Astro?** Follow one of our guides to migrate from a different project/framework.
+
+## Migration Guides
+
+<CMSGuidesNav />
+
+Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!
+
+## Why migrate your site to Astro?
+
+Astro provides many benefits: performance, simplicity and several built-in and available integrations!
```

Comment:
```suggestion
Astro provides many benefits: performance, simplicity, and many of the features you want built right in to the framework. When you do need to extend your site, Astro provides several official and 3rd-party community integrations!
```

Not entirely sold on this, but if there's a way to say "some of the stuff you have to install separately (and you're almost certainly going to anyway) is just included in Astro. Plus, there's more"... this is what I was trying to get at.